processing/src/main/java/io/druid/segment/StorageAdapter.java

```diff
@@ -54,15 +54,16 @@
   @Nullable
   Comparable getMaxValue(String column);
   Capabilities getCapabilities();
+  boolean areDimensionValuesSorted();
 
   /**
-   * Returns capabilities of a particular column, if known. May be null if the column doesn't exist, or if
-   * the column does exist but the capabilities are unknown. The latter is possible with dynamically discovered
+   * Returns storageAdapter of a particular column, if known. May be null if the column doesn't exist, or if
```

Comment:
Please revert javadoc as well.

Reply:
Probably `@Deprecated`?

Reply:
Oops, that change was overzealous intellij